src/tsickle.ts

```diff
@@ -300,7 +300,7 @@ class Annotator extends ClosureRewriter {
    *     emit it as is and visit its children.
    */
   maybeProcess(node: ts.Node): boolean {
-    if (node.flags & ts.NodeFlags.Ambient) {
+    if ((node.flags & ts.NodeFlags.Ambient) || /\.d\.ts$/.test(this.file.fileName)) {
```

Comment:
extract to a helper isDeclarationFile, to resist writing the regexp on multiple occasions.